singularity-eos/eos/eos_jwl.hpp

```diff
@@ -40,7 +41,16 @@ class JWL : public EosBase<JWL> {
   JWL() = default;
   PORTABLE_INLINE_FUNCTION JWL(const Real A, const Real B, const Real R1, const Real R2,
                                const Real w, const Real rho0, const Real Cv)
-      : _A(A), _B(B), _R1(R1), _R2(R2), _w(w), _rho0(rho0), _Cv(Cv) {}
+      : _A(A), _B(B), _R1(R1), _R2(R2), _w(w), _rho0(rho0), _Cv(Cv),
+        _c1(robust::ratio(A, rho0 * R1)), _c2(robust::ratio(B, rho0 * R2)) {
+    assert(A != 0.0);
+    assert(B != 0.0);
```

Comment:
We don't need this restriction I don't think. Plus, these parameters can be negative so I think it's fine to leave them unconstrained.

Reply:
fixed